entity-server-api/src/main/java/org/terracotta/entity/ActiveServerEntity.java

```diff
@@ -104,6 +104,12 @@ default R invoke(ClientDescriptor descriptor, M message) throws EntityUserExcept
   void handleReconnect(ClientDescriptor clientDescriptor, byte[] extendedReconnectData) throws ReconnectRejectedException;
 
   /**
+   * <p>Called when a reconnect event has finished. Either all clients have reconnected, or
+   * the reconnect window has timed out.</p>
+   */
+  void reconnectFinished();
```

Comment:
Can you add to the Javadoc something about how this is called in relation to invokeActive()?

In particular, I would expect that invokeActive() won't be called until after reconnectFinished() is called.